src/Billing/Utilities/PayPalIpnClient.cs

```diff
@@ -30,6 +32,7 @@ public async Task<bool> VerifyIpnAsync(string ipnBody)
             Method = HttpMethod.Post,
             RequestUri = _ipnUri
         };
+        _httpClient.DefaultRequestHeaders.Add("User-Agent", "CSharp-IPN-VerificationScript");
```

Comment:
It could be useful to include additional info as well, just at the start of the method (GitHub doesn't let me comment on a line that hasn't been changed, so apologies for leaving this comment here!):

```C#
_logger.LogInformation("Verifying IPN with PayPal at {Timestamp}: {VerificationUri}", DateTime.UtcNow, _ipnUri);
```